dashboard/src/main/resources/templates/component.ftl

```diff
@@ -110,8 +110,28 @@
     </#if>
 
     <h2>Dependencies</h2>
-    <pre class="dependency-tree">${dependencyTree}</pre>
-    
+
+    <#macro formatDependencyNode tree currentNode parent>
+      <#-- tree: ListMultimap<DependencyPath, DependencyPath>, currentNode: DependencyPath -->
```

Comment:
Ideally we shouldn't need to pass in currentNode from Java. Can we just create it here instead?

Reply:
FreeMarkers' ObjectConstructor worked.
https://freemarker.apache.org/docs/api/freemarker/template/utility/ObjectConstructor.html